### ISSUE TITLE
No Way to Reliably Trigger Right-click Context Menu from Keyboard

### ISSUE THREAD

**Issue by BlindGuyNW** [MegaMek/mekhq#2418]
### Environment
What version of MekHQ does your issue apply to? 0.47.17
What operating system are you using?  WIndows
What java version are you using?  1.8.0_281

### Description
User Impact
It is impossible to reliably trigger the right click context menu from the keyboard. This makes certain tasks such as assigning personnel difficult to impossible with a screen reader.

Repro Steps:
1. Activate MekHQ.
2. Load a campaign with an existing roster for simplicity.
3. Move to the personnel tab.
4. Move into the personnel table with Tab and go down a few rows as appropriate.
5. Attempt to open the context menu from the keyboard. Press shift+F10 and the applications menu key.

Actual Results
Shift+f10 opens the Windows "System," menu, with "minimize," "restore," etc. The application key appears to do nothing on my system, though this may be specific to my setup. In any case both shortcuts should ideally be supported.

Expected Results
A way should be provided to open this menu from the keyboard reliably across all platforms.

Possible Resolutions:
Add a shortcut such that shift+f10 opens the context menu on the selected item in keyboard focus, much as right click does for the mouse.

### Files

NA
